app/Components/Services/PasswordResetService.php

```diff
@@ -9,26 +9,28 @@ class PasswordResetService
 {
     private $model;
 
-    public function __construct(PasswordReset $reset, TokenGenerator $tokenGenerator)
+    public function __construct(PasswordReset $reset)
     {
         $this->model = $reset;
-        $this->generator = $tokenGenerator;
     }
 
     public function queryByEmail(string $email)
     {
         return $this->model->where('email', $email);
     }
 
+    public function find(array $params)
+    {
+        return $this->model->where($params)->first();
+    }
+
     public function findByEmail(string $email)
     {
         return $this->queryByEmail($email)->first();
     }
 
-    public function storePasswordResetRequest(string $email)
+    public function storePasswordResetRequest(string $email, string $token)
```

Comment:
Isso tá me parecendo um repository, pq apenas tá abstraindo algumas querys. Um service é usado, até onde eu sei, para executar um conjunto de regras de negócio que o controller não deveria fazer.

Reply:
Opa, obrigado, pus o nome indiscriminadamente sem  pensar em padrões porque de início a intenção era só testar, a partir dos testes se tornaram um repositório. Valeu por lembrar, vou alterar aqui.